components/AgreementList/tokenConfig.js

```diff
@@ -0,0 +1,23 @@
+// tokenConfig.js
+
+import tokens from "../../public/allowedTokens.json";
+
+const ENV = process.env.NODE_ENV;
+const FAKE_SOL = process.env.NEXT_PUBLIC_SOLANA_FAKE_STABLE_ADDRESS;
+const FAKE_EVM = process.env.NEXT_PUBLIC_FAKE_STABLE_ADDRESS;
+
+export const getTokens = (chain) => {
+    if (ENV !== 'development') {
```

Comment:
this should not be here. There must be a function `getTokens` on each chain implementation that returns the token list. We can't have any chain specific stuff in the common interface.